src/client/components/MealList.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+
+function MealList() {
+  const [meals, setMeals] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const res = await fetch("api/meals");
+      const data = await res.json();
+      setMeals(data.data);
+    })();
+  }, []);
+
+  return (
+    <div>
+      <h2>All Meals</h2>
+      {meals.map((meal) => (
```

Comment:
Consider the case where the meals are not fetched, so that this does not throw an error 💡